Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/S3Express/S3ExpressIdentity.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import protocol SmithyIdentityAPI.Identity
+import struct Foundation.Date
+
```

Comment:
A type to hold a S3 Express identity.

Reply:
Though this type is more or less the same as `AWSCredentialIdentity` (minus only the account ID), a separate type is used for S3 Express credentials because the two credential types are not interchangeable, and separate types helps make clear which credentials are being provided.